pkg/kubevip/config_envvar.go

```diff
@@ -24,6 +24,9 @@ const (
 	//vipInterface - defines the interface that the vip should bind too
 	vipInterface = "vip_interface"
 
+	//vipInterface - defines the interface that the vip should bind too
+	vipServicesInterface = "vip_servicesinterface"
```

Comment:
From my perspect, services_vip_interface would be better than vip_servicesinterface. What do you think?